test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# manually exiting from script, because after-build needs to run always
+set +e
+echo "1. Running before-build"
+./cc-test-reporter before-build
```

Comment:
Try adding `CC_TEST_REPORTER_ID=${CC_TEST_REPORTER_ID}` before this line, I wonder if that would make a difference?  Or maybe we just check it's not empty before running the before and after?

Reply:
Added that env to github action workflows.